corehq/apps/data_interfaces/migrations/0032_bootstrap_audit_events_for_update_rules.py

```diff
@@ -0,0 +1,28 @@
+from django.core.management import call_command
+from django.db import migrations
+
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def bootstrap_audit_events_for_automatic_update_rule(apps, schema_editor):
+    call_command('bootstrap_field_audit_events',
+                 'top-up',
+                 ['AutomaticUpdateRule'])
+
+
+def noop(*args, **kwargs):
+    pass
```

Comment:
Alternatively, there is a `RunPython.noop`.

```suggestion
```

Reply:
See a45dcff40637a9d9970b319920a7835ff0cb04fd